woods_hole_obs_data/project_metadata.csv

```diff
@@ -54,3 +54,4 @@ project_name,contributor_name,project_title,project_summary,catalog_xml
 "RCNWR","N. Ganju","Rachel Carson NWR 2013","Oceanographic and water-quality observations made at 3 sites in the wetlands of the Rachel Carson National Wildlife Refuge.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/RCNWR/catalog.xml"
 "BW2011","N. Ganju"," Blackwater 2011"," Oceanographic and Water-Quality Measurements made at several sites in 2 watersheds in Blackwater National Wildlife Refuge.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/BW2011/catalog.xml"
 "BARNEGAT_2014","N. Ganju"," Estuarine Physical Response to Storms (EPR), Barnegat Bay New Jersey, 2014","Oceanographic, and water-quality observations made at two locations in Barnegat Bay, NJ, 2014 to characterize the processes that affect sediment transport to tidal wetlands.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/BARNEGAT_2014/catalog.xml"
+"CALCSTWET","N. Ganju",California Coastal Wetlands, 2013-2014","Oceanographic and Water Quality Measurements in two Southern California Coastal Wetlands, 2013-2014","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/CALCSTWET/catalog.html"
```

Comment:
Dang, yes.  But when I try to go to fix it, I can't find it.  Where do you
click to find the modified code?

Thanks!

On Thu, Apr 7, 2016 at 3:01 PM, Rich Signell <notifications@github.com>
wrote:

> In woods_hole_obs_data/project_metadata.csv
> <https://github.com/USGS-CMG/usgs-cmg-portal/pull/186#discussion_r58926525>
> :
>
> > @@ -54,3 +54,4 @@ project_name,contributor_name,project_title,project_summary,catalog_xml
> >  "RCNWR","N. Ganju","Rachel Carson NWR 2013","Oceanographic and water-quality observations made at 3 sites in the wetlands of the Rachel Carson National Wildlife Refuge.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/RCNWR/catalog.xml"
> >  "BW2011","N. Ganju"," Blackwater 2011"," Oceanographic and Water-Quality Measurements made at several sites in 2 watersheds in Blackwater National Wildlife Refuge.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/BW2011/catalog.xml"
> >  "BARNEGAT_2014","N. Ganju"," Estuarine Physical Response to Storms (EPR), Barnegat Bay New Jersey, 2014","Oceanographic, and water-quality observations made at two locations in Barnegat Bay, NJ, 2014 to characterize the processes that affect sediment transport to tidal wetlands.","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/BARNEGAT_2014/catalog.xml"
> > +"CALCSTWET","N. Ganju",California Coastal Wetlands, 2013-2014","Oceanographic and Water Quality Measurements in two Southern California Coastal Wetlands, 2013-2014","http://geoport.whoi.edu/thredds/catalog/usgs/data2/emontgomery/stellwagen/Data/CALCSTWET/catalog.html"
>
> missing " before California?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/USGS-CMG/usgs-cmg-portal/pull/186/files/85f9eec39637b3cd0803989d10d38b7e22e6dafe#r58926525>
>



-- 
Ellyn Montgomery, Oceanographer and Data Manager
U.S. Geological Survey
Woods Hole Coastal and Marine Science Center
384 Woods Hole Rd., Woods Hole, MA, 02543-1598
(508) 457-2356